conftest.py

```diff
@@ -0,0 +1,54 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
```

Comment:
-2018

Reply:
@tgamblin Should we change manually the copyright date, or do you plan updating all the copyrights at once (like last year)?

Reply:
Let's do it all at once.  Or better yet I would love to remove stuff like this from headers that just causes commit noise... would be nice to use SPDX headers but I'm not sure the lab has decided that's ok yet.  I am also not sure if that fixes the copyright.